src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -273,5 +279,14 @@ public override ParserBase DecorateCodeParser([NotNull] ParserBase incomingCodeP
                     CreateModelExpressionMethodName = CreateModelExpressionMethod
                 });
         }
+
+        private IReadOnlyList<ChunkTree> GetInheritedChunkTrees(string sourceFileName)
+        {
+            var inheritedChunkTrees = GetInheritedChunkTreeResults(sourceFileName)
+                .Select(result => result.ChunkTree)
+                .ToList();
```

Comment:
Just following what we did before. @pranavkm was there a specific reason for `IReadOnlyList` (I can't recall)?